pymc3/gp/cov.py

```diff
@@ -34,6 +36,7 @@
     "Coregion",
     "ScaledCov",
     "Kron",
+    "Exponentiated",
```

Comment:
Since `Exponentiated` isn't necessarily user-facing because you added `__pow__`, it probably shouldn't be included in `__all__`.